c/include/ml-api-service.h

```diff
@@ -224,6 +224,102 @@ int ml_service_query_create (ml_option_h option, ml_service_h *handle);
  */
 int ml_service_query_request (ml_service_h handle, const ml_tensors_data_h input, ml_tensors_data_h *output);
 
+/**
+ * @brief Creates ml remote service handle with given ml-option handle.
+ * @details The caller should set one of "remote_sender" and "remote_receiver" as a service type in @a ml_option.
+ * @remarks The @a handle should be destroyed using ml_service_destroy().
+ * @param[in] option The option used for creating query service.
+ * @param[out] handle Newly created query service handle is returned.
+ * @return @c 0 on Success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ * @retval #ML_ERROR_STREAMS_PIPE Failed to launch the pipeline.
+ * @retval #ML_ERROR_TRY_AGAIN The pipeline is not ready yet.
+ */
+int ml_remote_service_create (ml_option_h option, ml_service_h *handle);
```

Comment:
This change requires new ACR, plz do not add public APIs in this phase. Requesting remote service is next topic and it requires more discussion, e,g, net connections and sending files via devices.

Reply:
Thanks, they are moved do `ml-api-service-private.h`.